### ISSUE TITLE
No longer working

### ISSUE THREAD

**Issue by LordFrz** [bozbez/win-capture-audio#261]
This worked awesomely until a few months ago. Nothing else works as well as this so it's sad to see. I'm on Windows 10, which seems to have broken it with an update.

**Comment by OGNetheri** [comment]
Check that your sound channels are set to "stereo" in OBS settings. For my experience this plugin can't capture multichannels. Also check that you don't accidentaly use OBS's native BETA version of the source (there reads "BETA" in the source options, while plugin doesn't have BETA marking"). OBS's own capture for some reason uses the executables definition name instead of the exe's name and that sometimes prevents the capturing with it.

**Comment by JustCallMeQuincy** [comment]
I'm on Windows 10 and everything works splendidly for me so far

**Comment by jarrard** [comment]
yeah it still works it seems.  I had 5.1 set and it didn't like. Would be nice if it could support 5.1 but ell well.
